### PR TITLE
Bug 1139470 - Minimized and maximized statusbar can become out of sync

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

File: apps/system/test/unit/statusbar_test.js [hunk 1]
```diff
@@ -876,6 +876,7 @@ suite('system/Statusbar', function() {
     });
 
     test('sheets-gesture-end', function() {
+      StatusBar._pausedForGesture = true;
```
Comment (etiennesegonzac) [inline apps/system/test/unit/statusbar_test.js]: please do not set internal variables from tests, keep the tests "externals" you probably want to dispatch a `sheets-gesture-begin` instead.

It keeps the tests more robust (ie. won't have to change the test if we change the internal variable name) and closer to what really happens.

File: apps/system/test/unit/statusbar_test.js [hunk 2]
```diff
@@ -926,6 +927,29 @@ suite('system/Statusbar', function() {
         testEventThatResumeIfNeeded.bind(this)('homescreenopened');
       });
     });
+
+    suite.only('edge swipe should resume symmetrically', function() {
+      test('with many begin events', function() {
+        StatusBar._pausedForGesture = false;
```
Comment (etiennesegonzac) [inline apps/system/test/unit/statusbar_test.js]: same comment, you probably just remove the line

File: apps/system/test/unit/statusbar_test.js [hunk 3]
```diff
@@ -926,6 +927,29 @@ suite('system/Statusbar', function() {
         testEventThatResumeIfNeeded.bind(this)('homescreenopened');
       });
     });
+
+    suite.only('edge swipe should resume symmetrically', function() {
```
Comment (etiennesegonzac) [inline apps/system/test/unit/statusbar_test.js]: please don't forget the `.only` here

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3f5183ab940e732c7c90002f165c97aac502039a

**Pull request merged**